templates/_components/index.html

```diff
@@ -447,13 +455,36 @@ <h3>Radio</h3>
       </div>
 
       <h3>Radio list</h3>
-      <p>Pass Django radio fields in to a list of radios</p>
+      <p>Pass Django choice fields in to a list of radios</p>
+      <ul>
+        <li><code>choices</code>: [dict] - dict of [value, label] for the options</li>
+        <li><code>class</code>: [string] - add HTML classes to the component</li>
+        <li><code>field</code>: a Django form field
+          <ul>
+            <li><a href="https://devdocs.io/django~4.1/ref/forms/api#django.forms.Form.auto_id"><code>auto_id</code></a></li>
+            <li><a href="https://devdocs.io/django~4.1/ref/forms/api#django.forms.Form.errors"><code>errors</code></a></li>
+            <li><a href="https://devdocs.io/django~4.1/ref/forms/api#django.forms.BoundField.html_name"><code>html_name</code></a></li>
+            <li><a href="https://devdocs.io/django~4.1/ref/forms/api#django.forms.BoundField.id_for_label"><code>id_for_label</code></a></li>
+            <li><a href="https://devdocs.io/django~4.1/ref/forms/api#django.forms.BoundField.label"><code>label</code></a></li>
+          </ul>
+        </li>
+        <li><code>hint_text</code>: [string] - add a subtitle as hint text for the select</li>
+        <li><code>id</code>: [string] - set the HTML id for the select element</li>
+        <li><code>label</code>: [string] - overwrite the label passed from Django</li>
+        <li><code>name</code>: [string] - set the HTML name for the select element</li>
+        <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required"><code>required</code></a></li>
+        <li><code>selected</code>: [string] - matches value of choice for preselected option</li>
+      </ul>
+      <div class="not-prose flex flex-col gap-y-8 relative w-screen max-w-5xl left-1/2 -translate-x-1/2 p-8 bg-slate-200 border border-slate-500 rounded">
+        {% form_radios class="w-full max-w-lg mx-auto" label="Choose a language" required=True field=example_form.example_radios choices=example_form.example_radios.field.choices selected=example_form.example_radios.value %}
+        <pre><code class="language-django">{% form_radios class="w-full max-w-lg mx-auto" label="Choose a language" required=True field=example_form.example_radios choices=example_form.example_radios.field.choices selected=example_form.example_radios.value %}{% endverbatim %}</code></pre>
```

Comment:
```suggestion
        <pre><code class="language-django">{% verbatim %}{% form_radios class="w-full max-w-lg mx-auto" label="Choose a language" required=True field=example_form.example_radios choices=example_form.example_radios.field.choices selected=example_form.example_radios.value %}{% endverbatim %}</code></pre>
```

Missing the opening tag here